js/modules/timeline.src.js

```diff
@@ -100,19 +100,19 @@ undocumentedSeriesType('timeline', 'line',
              */
             connectorColor: '${palette.neutralColor100}',
```

Comment:
The connectorColor should default to the point.color || series.color like in pie charts. https://api.highcharts.com/highcharts/series.pie.dataLabels.connectorColor

Reply:
> There's a code block where oldTextWidth is deleted. This looks like a workaround. Check if it is still necessary after 6d6773e.

Good point. It's currently not necessary.
> In bindAxes, why force axis options? Maybe it’s better (more predictable to users) to just set xAxis.visible: false and yAxis.visible: false in the demos?

Agree, it would be more predictable to hide axes on demos.
> We have many cases where we borrow functions from the core series types, so I think we should continue doing that. In the (near) future we may have a build system that allows us to borrow functions without importing the whole parent series, but then we should change all cases consistently. Alternatively, we can create a mixin now.

Those two functions (fro Pie and Timeline) are not identical, because it's necessary to call `processData()` before redrawing the chart. We can say, that function is taken from Pie, but it's a bit extended and it's not possible to wrap it. My proposition is to do something like below:

```js
        setVisible: function (vis, redraw) {
            var point = this,
                series = point.series;

            redraw = pick(redraw, series.options.ignoreHiddenPoint);

            seriesTypes.pie.prototype.pointClass.prototype
                .setVisible.call(point, vis, false);
            // Process new data
            series.processData();

            if (redraw) {
                series.chart.redraw();
            }
        },
```
Regarding connector groups, currently they're moved to a specific data label's group, and their default colors are taken from the point options.